setup.py

```diff
@@ -30,13 +30,20 @@
 
 long_description = open("README.rst").read()
 
-install_requires = ['pymongo>=3.6,<4']
+install_requires = ['pymongo>=3.7']
 
 tests_require = ['mockupdb>=1.4.0']
 
 if sys.version_info[0] < 3:
     install_requires.append('futures')
 
+# Optional dependency with a minimum version
+try:
+    import tornado
+    install_requires.append('tornado>=4.0')
+except ImportError:
+    pass
```

Comment:
Why do we only include tornado as a dependency when the build environment includes tornado? See my comment above for why I don't think this will work as expected for wheels -- universal or otherwise.